include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp

```diff
@@ -83,18 +87,30 @@ class Orbit : public Trajectory
     /// @brief Constructor.
     ///
     /// @param aModel An orbit model.
-    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body.
-    Orbit(const orbit::Model& aModel, const Shared<const Celestial>& aCelestialObjectSPtr);
+    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body. Defaults to Earth.
+    Orbit(
+        const orbit::Model& aModel,
+        const Shared<const Celestial>& aCelestialObjectSPtr = std::make_shared<Celestial>(Earth::Default())
+    );
 
     /// @brief Constructor.
     ///
     /// @param aStateArray An array of states.
     /// @param anInitialRevolutionNumber An initial revolution number.
-    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body.
+    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body. Defaults to Earth.
     Orbit(
         const Array<State>& aStateArray,
         const Integer& anInitialRevolutionNumber,
-        const Shared<const Celestial>& aCelestialObjectSPtr
+        const Shared<const Celestial>& aCelestialObjectSPtr = std::make_shared<Celestial>(Earth::Default())
+    );
+
+    /// @brief Constructor.
+    ///
+    /// @param aTLE Two-Line Element (TLE).
+    /// @param aCelestialObjectSPtr A shared pointer to a central celestial body. Defaults to Earth.
+    Orbit(
+        const TLE& aTLE,
+        const Shared<const Celestial>& aCelestialObjectSPtr = std::make_shared<Celestial>(Earth::Default())
```

Comment:
I think it might make more sense to make this a factory method instead of another constructor up here, because its specific to `TLE` models, and duplicates the functionality offered by the first constructor
```
Orbit::FromTLE(
	const TLE& aTLE,
)
```
Since you'd add this convenience factory method above, do you also want to add one for a Keplerian model and a Propagated model? Cause if not I don't see the need to prioritize creating one for TLE over the others, since the construction of an orbit from a TLE is already quite simple

Reply:
Agreed that we can make it a factory method. I would then remove the state based Orbit constructor as well and do a `fromTabulated`